perfmetrics/scripts/ml_tests/tf/resnet/setup_scripts/setup_container.sh

```diff
@@ -17,7 +17,7 @@ cd -
 
 # Mount the bucket and run in background so that docker doesn't keep running after resnet_runner.py fails
 echo "Mounting the bucket"
-nohup gcsfuse/gcsfuse --foreground --implicit-dirs --enable-storage-client-library --debug_fs --debug_gcs --max-conns-per-host 100 --log-format "text" --log-file /home/logs/gcsfuse.log --stackdriver-export-interval 60s ml-models-data-gcsfuse myBucket > /home/output/gcsfuse.out 2> /home/output/gcsfuse.err &
```

Comment:
We are adding --debug_fuse flag instead of this?

Reply:
Thanks, added --debug_fuse flag.